ui/app/appointments/app.js

```diff
@@ -79,7 +79,9 @@ angular
                 url: '/calendar',
                 tabName: 'calendar',
                 params: {
-                    viewDate: null
+                    viewDate: null,
+                    getAppointments: true,
```

Comment:
I don't like this naming: `getAppointments`. Prefixing something with `get` means this is a getter function. However this looks like a flag telling whether appointments should be fetched, am I correct?

Reply:
Maybe `doFetchAppointments` or `doFetchAppointmentsData`.